src/relayclient/ClientWinstonLogger.ts

```diff
@@ -0,0 +1,63 @@
+import Cookies from 'js-cookie'
```

Comment:
why do you have a separate server and client logger? looks like duplicate code.
2. Winston is just an implementation issue, internal to the logger, so don't put it in the file name name

Reply:
The config is quite different, in face. local storage and stuff.

As I've said, this particular file creates this particular logger instance. Makes sense to me.